server/utils/utils.py

```diff
@@ -0,0 +1,32 @@
+import contextlib
+import errno
+import socket
+
+def find_available_port(host, port):
+    """
+    Helper method to find open port on host. Tries 50 ports incremented from the specified port
+    """
+    new_port = port
+    found_port = False
+    num_ports_to_try = 50
+    with contextlib.closing(socket.socket(socket.AF_INET, socket.SOCK_STREAM)) as s:
+        for port_to_try in range(port, port + num_ports_to_try):
+            while True:
+                try:
+                    s.bind((host, port_to_try))
+                    found_port = True
```

Comment:
at this point, I believe you can just `return port_to_try`.

you can remove `found_port`, and the associated logic with that.

Reply:
yes, yes I can. 👍